src/app/pages/UpdateDestination/UpdateDestination.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react';
+import CardTitle from '../../components/CardTitle/CardTitle';
+import DestinationForm from '../../components/DestinationForm/DestinationForm';
+import InputField from '../../components/InputField/InputField';
+import InputLabel from '../../components/InputLabel/InputLabel';
+import DefaultButton from '../../components/DefaultButton/DefaultButton';
+import type { DestinationType } from '../../utils/DestinationType';
+import type { GeoJsonType } from '../../utils/GeoJsonType';
+import MapWithMarker from '../../components/MapWithMarker/MapwithMarker';
+import { useNavigate } from 'react-router';
+
+export default function UpdateDestination(): JSX.Element {
+  const currentId = window.localStorage
+    .getItem('UpdateId')
+    ?.replaceAll('"', '');
+
+  const currentDestination: any[] = JSON.parse(
+    window.localStorage.getItem('destination') as string
+  );
```

Comment:
I think you still have that old `useLocalStorage` hook lying around somewhere in the attic, I suggest using it :)